src/main/java/oop/automobiles/ConcreteTruck.java

```diff
@@ -0,0 +1,21 @@
+package oop.automobiles;
+
+/*
+ * A class which defines all Concrete Trucks.
+ */
+public final class ConcreteTruck extends Truck {
+ 
+    // Constructor method for ConcreteTruck
+    public ConcreteTruck(String license, int wheels) {
+        super(license, wheels);
+    }
+
+    public void mixConcrete(String item) {
+        this.getCargo().add(item);  // Adds items to the cargo list (loads and mixes concrete)
+    }
+
+    public void carryConcrete(String item) {
```

Comment:
be careful with methods like this - there's no apparent use of the String item argument, and the method name doesn't necessarily indicate that the cargo would be cleared despite your provided comment.